docs/src/assets/js/search.js

```diff
@@ -164,7 +164,7 @@ document.addEventListener('keydown', function (e) {
     if ((e.metaKey || e.ctrlKey) && e.key === 'k') {
         e.preventDefault();
         searchInput.focus();
-        document.querySelector('.search').scrollIntoView({ behaviour: "smooth", block: "start" });
+        document.querySelector('.search').scrollIntoView({ behavior: "smooth", block: "start" });
```

Comment:
Yeah, [it should be behavior](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#examples)